src/genie/libs/parser/iosxe/tests/ShowPolicyMap/cli/equal/golden_output16_output.txt

```diff
@@ -0,0 +1,7 @@
+ Policy Map map1
+    Class cs1
+      priority 9 (%)
+    Class cs2
+      bandwidth 9 (%)
```

Comment:
Bandwidth value is not getting captured in your output, kindly work on regex p7 as well. This [regex](https://regex101.com/r/X2OXmT/1) may be suitable for this.

Reply:
updated output to focus on priority, previous bandwidth included as a mistake taken from working examples.